forest/src/cli/wallet_cmd.rs

```diff
@@ -165,10 +165,23 @@ impl WalletCommands {
             }
             Self::List => {
                 let response = wallet_list().await.map_err(handle_rpc_err).unwrap();
+                let default = wallet_default_address()
+                    .await
+                    .map_err(handle_rpc_err)
+                    .unwrap();
 
-                response.iter().for_each(|address| {
-                    println!("{}", address.0);
-                });
+                println!("Address                                    Default  Balance");
+
+                for address in response {
+                    let addr = address.0.to_string();
+                    let def = if addr == default { "X" } else { "" };
+                    let bal = wallet_balance((addr.clone(),))
+                        .await
+                        .map_err(handle_rpc_err)
+                        .unwrap();
+
+                    println!("{}  {:7}  {}", addr, def, bal);
```

Comment:
I would like to see this to be aligned for sure with the header. E.g. `{addr:<the entire length of address column as in header>}{def:<the same>}{bal:<the same>}`. Right now, looking at the code it's hard to say if all will fit.